builtin/providers/aws/resource_aws_launch_configuration.go

```diff
@@ -206,3 +461,9 @@ func resourceAwsLaunchConfigurationDelete(d *schema.ResourceData, meta interface
 
 	return nil
 }
+
+func readBlockDevicesFromLaunchConfiguration(d *schema.ResourceData, launchConfiguration *autoscaling.LaunchConfiguration, autoscalingconn *autoscaling.AutoScaling) map[string]interface{} {
+	var blockDevices = make(map[string]interface{})
+	// Need to figure out how to determine the various instance types.
```

Comment:
I believe I already did that. But there is a read section that I don't exactly know how to follow.